make/autoconf/flags-ldflags.m4

```diff
@@ -71,9 +71,6 @@ AC_DEFUN([FLAGS_SETUP_LDFLAGS_HELPER],
       LIBJSIG_HASHSTYLE_LDFLAGS="-Wl,--hash-style=both"
     fi
 
-    # Add -z defs, to forbid undefined symbols in object files.
-    BASIC_LDFLAGS="$BASIC_LDFLAGS -Wl,-z,defs"
```

Comment:
Removing this seems like it just papers over the problem and would also allow for undefined symbols to creep in.  The commit says the missing symbol was 'environ', can you provide more context?  Was that in linking the VM?